app/src/main/kotlin/com/wire/android/ui/home/newconversation/common/GroupButton.kt

```diff
@@ -0,0 +1,60 @@
+package com.wire.android.ui.home.newconversation.common
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.wire.android.R
+import com.wire.android.ui.common.button.IconAlignment
+import com.wire.android.ui.common.button.WireTertiaryButton
+import com.wire.android.ui.common.button.wireSecondaryButtonColors
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.common.textfield.WirePrimaryButton
+import com.wire.android.ui.theme.wireDimensions
+
+@Composable
+fun GroupButton(groupSize: Int) {
+    Row(
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .height(82.dp)
```

Comment:
I feel like I have been seeing this magic `82.dp` number way too many times. Can we extract it to a wire dimension value?

Reply:
if it has nothing to do with the send button of the Message Composer, can we use `spacing80x` instead?